########################
## Hardware Interface ##
########################
AG95:
  hardware_interface:
    loop_hz: 1
    joints:
      - 'AG95_finger_joint'
